plugin/pkg/admission/initialization/initialization.go

```diff
@@ -208,6 +208,12 @@ func (i *initializer) Admit(a admission.Attributes) (err error) {
 		}
 		updated := accessor.GetInitializers()
 
+		// allow administrators to bypass initialization by setting an empty initializers struct
+		if updated != nil && len(updated.Pending) == 0 && updated.Result == nil {
```

Comment:
I am a little concerned that this logic leaves a landmine should anyone ever add a field to the initializers section. Not a problem you added, obviously. Also no great way to address it. Putting the `len(updated.Pending) == 0 && updated.Result == nil` into a `EmptyInitializers(initializers) bool` predicate (and finding all places that are repeating this logic) might help a bit. Not required for this PR though.

Reply:
I started moving the 2 from this file into a function EmptyInitializers, but then I realized that the logic is in lots of other files as well. At least [this](https://github.com/kubernetes/kubernetes/blob/440884be9ae54a500139697890889e89dabe25cf/staging/src/k8s.io/apimachinery/pkg/util/initialization/initialization.go) one and [this](https://github.com/kubernetes/kubernetes/blob/12d73c31a97188286e7f4c8a3aec7fcd38a76a83/cmd/kube-apiserver/app/testing/server_test.go) one but possibly more

Reply:
Wait, that first one looks like a predicate, but it's in a totally different directory